doc/api/process.md

```diff
@@ -1793,6 +1793,22 @@ added: v11.8.0
 reports for the current process. Additional documentation is available in the
 [report documentation][].
 
+### `process.report.compact`
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
```

Comment:
I don't think this needs to be experimental on master anymore. It will be if it's backported though.